Makefile

```diff
@@ -440,6 +445,7 @@ TEST_WEB_74 := \
 	test_web_symfony_42 \
 	test_web_symfony_44 \
 	test_web_wordpress_48 \
+	test_web_wordpress_55 \
```

Comment:
Just to confirm: does WordPress 5.5 not work on PHP 8?

Reply:
It does not work as it is (unexpected status codes in case of missing and missing spans). It seems that the [version expected to be working with PHP 8 is 5.6](https://make.wordpress.org/core/2020/10/06/call-for-testing-php-8-0/) .

Reply:
Perfect. Thanks for looking into this.